addons/account/static/src/components/x2many_buttons/x2many_buttons.js

```diff
@@ -47,9 +47,49 @@ class X2ManyButtons extends Component {
     }
 }
 
+class X2ManyMultipleModels extends X2ManyButtons {
+    /**
+     * This extension allows the use of X2ManyButtons when dealing with
+     * form views of different models. actionName is a function that
+     * returns the appropriate action when it receives one or multiple ids.
+     */
+    static props = {
+        ...standardFieldProps,
+        actionName: { type: String },
+    };
+
+    setup() {
+        this.orm = useService("orm");
+        this.action = useService("action");
+    }
+
+    async openFormAndDiscard(id) {
+        const action = await this.orm.call(this.currentField.resModel, this.props.actionName, [id], {});
```

Comment:
I tried making this class reusable, by making the action name and the model name dynamic, but we can discuss if this is the best approach